src/core_plugins/kibana/public/visualize/wizard/step_1.html

```diff
@@ -27,6 +27,7 @@ <h1 class="kuiTitle kuiVerticalRhythm kuiVerticalRhythm--medium">
         type="text"
         placeholder="Search visualization types..."
         ng-model="searchTerm"
+        role="search"
```

Comment:
`role=search` is unfortunately a little bit tricky. If you put it on the `input` you will overwrite it's native role as a textfield, which will cause weird behavior in assistive technologies (like screen readers). Also you shouldn't put it to the form, since it would cause the same. Ususally it makes sense to put it on a `div` inside the form, that contains the `input`. In this case, this can be easily achieved, since we already have an encapsulated `div` (the one also commented with `Search`. Please put the role there.

There is an ongoing discussion in the ARIA standard, in [several](https://github.com/w3c/wcag/issues/113) [different](https://github.com/w3c/html-aria/issues/18) [issues](https://github.com/w3c/aria/issues/85)... also see [Where to put your search role](http://adrianroselli.com/2015/08/where-to-put-your-search-role.html).